libsolidity/formal/SymbolicState.cpp

```diff
@@ -130,17 +130,22 @@ smtutil::Expression SymbolicState::txConstraints(FunctionDefinition const& _func
 {
 	smtutil::Expression conj = smt::symbolicUnknownConstraints(m_tx.member("block.coinbase"), TypeProvider::uint(160)) &&
 		smt::symbolicUnknownConstraints(m_tx.member("block.chainid"), TypeProvider::uint256()) &&
+		smt::symbolicUnknownConstraints(m_tx.member("block.coinbase"), TypeProvider::address()) &&
```

Comment:
Isn't this a duplicate of what's 2 lines above?

Reply:
ah, seems so ;) I thought I had missed it